.github/workflows/api-waf-plan.yml

```diff
@@ -4,6 +4,7 @@ on:
   pull_request:
     paths:
       - .github/workflows/api-waf-plan.yml
+      - .github/workflows/api-waf-apply.yml
```

Comment:
The apply workflow doesn't need to be included here

Reply:
Yea, this was only to kick off the initial plan runs because it wasn't picking up on the first commit that only included the apply changes, should have removed once I started changing other files